typescript/packages/plugins/ionic/src/address.ts

```diff
@@ -0,0 +1,5 @@
+export const ionicAddress = {
```

Comment:
This is not being used, remove?

Reply:
yes , i had created it long back, Now removed